content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -2332,6 +2332,114 @@ Starting in v1.16, this annotation was removed in favor of
 [Pod Priority](/docs/concepts/scheduling-eviction/pod-priority-preemption/).
 {{< /note >}}
 
+### jobset.sigs.k8s.io/jobset-name
+
+Type: Label
+
+Example:  `jobset.sigs.k8s.io/jobset-name: "my-jobset"`
+
+Used on: Jobs, Pods
+
+This label is used to store the name of the JobSet to which a Job or Pod belongs.
+JobSet is an extension API that you can deploy into your Kubernetes cluster.
+
+### jobset.sigs.k8s.io/replicatedjob-replicas
+
+Type: Annotation
+
+Example: `jobset.sigs.k8s.io/replicatedjob-replicas: "5"`
+
+Used on: ReplicatedJobs
+
+This annotation specifies the number of replicas for a ReplicatedJob.
+ReplicatedJob is an extension API that you can deploy into your Kubernetes cluster.
+
+### jobset.sigs.k8s.io/replicatedjob-name
+
+Type: Label
+
+Example: `jobset.sigs.k8s.io/replicatedjob-name: "my-replicatedjob"`
+
+Used on: Jobs
+
+This label is used to index into a Job's labels and retrieve the name of the parent ReplicatedJob.
+ReplicatedJob is an extension API that you can deploy into your Kubernetes cluster.
+
+### jobset.sigs.k8s.io/job-index
+
+Type: Annotation
+
+Example: `jobset.sigs.k8s.io/job-index: "0"`
+
+Used on: Jobs
+
+This annotation stores the index of the Job within the ReplicatedJob.
+ReplicatedJob is an extension API that you can deploy into your Kubernetes cluster.
+
+### jobset.sigs.k8s.io/job-key
+
+Type: Label
+
+Example: `jobset.sigs.k8s.io/job-key: "my-job-key"`
+
+Used on: Jobs
+
+This label provides a unique key for the job.
+
+### alpha.jobset.sigs.k8s.io/exclusive-topology
+
+Type: Annotation
+
+Example: `alpha.jobset.sigs.k8s.io/exclusive-topology: "zone"`
+
+Used on: JobSets, ReplicatedJobs
+
+You can set this annotation on a JobSet or a ReplicatedJob template to ensure exclusive job placement per topology group.
+JobSet and ReplicaSet are extension APIs that you can deploy into your Kubernetes cluster.
+
+### alpha.jobset.sigs.k8s.io/node-selector
+
+Type: Annotation
+
+Example: `alpha.jobset.sigs.k8s.io/node-selector-strategy=true`
+
+Used on: JObs ,Pods
+
+This annotation can be applied to a JobSet. When it's set, the JobSet controller modifies the Jobs and their corresponding Pods by adding node selectors. This ensures exclusive job placement per topology domain, restricting the scheduling of these Pods to specific nodes based on the strategy.
+
+### alpha.jobset.sigs.k8s.io/namespaced-job
+
+Type: Annotation, Label
+
+Example: `alpha.jobset.sigs.k8s.io/namespaced-job:my-jobset-replicatedjob-A-0`
```

Comment:
```suggestion
Example: `alpha.jobset.sigs.k8s.io/default_myjobset-replicatedjob-0`
```

see https://github.com/kubernetes-sigs/jobset/blob/main/pkg/controllers/jobset_controller.go#L804

Also, the format of child job names for JobSets is: `{jobset}-{replicatedJob}-{jobIndex}`